wrappers/python/python.hpp

```diff
@@ -148,7 +148,7 @@ MAP_FMT_TO_TYPE(RS2_FORMAT_Y8I, uint8_t);
 //MAP_FMT_TO_TYPE(RS2_FORMAT_INZI, );
 MAP_FMT_TO_TYPE(RS2_FORMAT_INVI, uint8_t);
 //MAP_FMT_TO_TYPE(RS2_FORMAT_W10, );
-
+MAP_FMT_TO_TYPE(RS2_FORMAT_FG, uint16_t);
```

Comment:
Why `uint16_t`? I see above that the other formats are `uint8_t`?

Reply:
its like MAP_FMT_TO_TYPE(RS2_FORMAT_Z16, uint16_t);